tests/test_licenses.py

```diff
@@ -11,47 +11,46 @@
 import nf_core.create
 import nf_core.licences
 
-
-class WorkflowLicensesTest(unittest.TestCase):
-    """A class that performs tests on the workflow license
-    retrieval functionality of nf-core tools."""
-
-    def setUp(self):
-        """Create a new pipeline, then make a Licence object"""
-        # Set up the schema
-        self.pipeline_dir = os.path.join(tempfile.mkdtemp(), "test_pipeline")
-        self.create_obj = nf_core.create.PipelineCreate("testing", "test pipeline", "tester", outdir=self.pipeline_dir)
-        self.create_obj.init_pipeline()
-        self.license_obj = nf_core.licences.WorkflowLicences(self.pipeline_dir)
-
-    def test_run_licences_successful(self):
-        console = Console(record=True)
-        console.print(self.license_obj.run_licences())
-        output = console.export_text()
-        assert "GPL v3" in output
-
-    def test_run_licences_successful_json(self):
-        self.license_obj.as_json = True
-        console = Console(record=True)
-        console.print(self.license_obj.run_licences())
-        output = json.loads(console.export_text())
-        for package in output:
-            if "multiqc" in package:
-                assert output[package][0] == "GPL v3"
-                break
-        else:
-            raise LookupError("Could not find MultiQC")
-
-    def test_get_environment_file_local(self):
-        self.license_obj.get_environment_file()
-        assert any(["multiqc" in k for k in self.license_obj.conda_config["dependencies"]])
-
-    def test_get_environment_file_remote(self):
-        self.license_obj = nf_core.licences.WorkflowLicences("methylseq")
-        self.license_obj.get_environment_file()
-        assert any(["multiqc" in k for k in self.license_obj.conda_config["dependencies"]])
-
-    @pytest.mark.xfail(raises=LookupError, strict=True)
-    def test_get_environment_file_nonexistent(self):
-        self.license_obj = nf_core.licences.WorkflowLicences("fubarnotreal")
-        self.license_obj.get_environment_file()
+# class WorkflowLicensesTest(unittest.TestCase):
```

Comment:
Maybe a TODO about that they need to be fixed or cut these out?

Reply:
Agree, adding a TODO in these cases is a good idea :+1:

Reply:
TODO sounds like a good compromise 👍🏽

Reply:
Added https://github.com/nf-core/tools/pull/1109/commits/25a52f8f294714b61cbb07a5874c8b74bb58b341